source/features/highlight-deleted-and-added-files-in-diffs.tsx

```diff
@@ -22,7 +22,7 @@ async function init(): Promise<void | false> {
 	const fileList = await elementReady([
 		'.toc-select details-menu[src*="/show_toc?"]', // `isPR`
 		'.toc-diff-stats + .content', // `isSingleCommit` and `isCompare`
-	].join(','));
+	].join(','), {stopOnDomReady: false});
```

Comment:
This should probably be fixed by listening for the `load` event of the `include-fragment` instead:

![image](https://user-images.githubusercontent.com/46634000/142758491-626ed2ff-4dd4-4140-a2ef-fee31fae52af.png)

This is likely the same solution as in #5066. Also there may be other features affected by this, so a reusable `onCompareDiffLoad` event listener might be useful.